homework-submission/main.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
+
+// The goal of this component is to take a parameter for list type as props
+// and insert a header with class of 'list-title' for lists
+const ListTitle = props => <h1 className="list-title">{props.type} List</h1>;
+
+// The goal of this component is to insert a list item with given description and
+// deadline, to give this inserted item proper class names, to add an event listener
+// that when a todo is clicked crosses it out as done if the todo is not done
+// and vice versa
+class ListItem extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { done: props.done ? props.done : false };
+    this.changeDoneState = this.changeDoneState.bind(this);
+  }
+
+  changeDoneState() {
+    if (this.state.done === true) {
```

Comment:
This code can be written shorter (and more readable). Hint: the if statement isn't needed